README.md

```diff
@@ -9,6 +9,7 @@ It attempts to provide the [feature request](https://github.com/wagtail/wagtail/
 Install the package
 ```
 pip install wagtail-orderable
+pip install wagtail-modeladmin
```

Comment:
Suggestion: I would say to install this before the package (i.e. put this statement before) as this is a dependency for this package, though it might be not so important as they will be installed